gradle.properties

```diff
@@ -1,3 +1,3 @@
-release_version=3.6.1
+release_version=3.6.2
```

Comment:
Maybe update minor version because codec will provide new public information after submitting this change